src/script/components/AvailabilityState.tsx

```diff
@@ -119,6 +121,12 @@ const AvailabilityState: React.FC<AvailabilityStateProps> = ({
       )}
     </React.Fragment>
   );
+
+  if (className) {
```

Comment:
Do you need the `if (className)` when using `clsx`?

Reply:
nope but this is solving some other issue, it adds a surrounding element based on this className